docs-v2/spec.yaml

```diff
@@ -486,6 +486,96 @@ paths:
                                 properties:
                                     message:
                                         type: string
+    /connection/metadata:
+        post:
+            description: Set custom metadata for the connection or connections. If updating multiple connections pass in an array of connection ids instead of a string.
+            parameters:
+                - name: provider_config_key
```

Comment:
Should we keep a query param in a POST? I feel is not standard and it's an opportunity to change it.

Reply:
Updated to a POST